packages/datasource-sparql/test/datasources/SparqlDatasource-test.js

```diff
@@ -104,8 +104,8 @@ describe('SparqlDatasource', function () {
     itShouldExecute(datasource, request,
       'a query for an object literal with newlines and quotes',
       { object: '"a\rb\nc"\r\n\\""', features: { quadPattern: true } },
-      'SELECT * WHERE {GRAPH ?g{?s ?p """a\rb\nc\\"\r\n\\\\\\""""}}',
```

Comment:
This test should not be changed.

Reply:
I agree but I do not see where they were coming from with the extra double quotes.
I ran it through a SPARQL validator (http://www.sparql.org/query-validator.html), and the two versions result in an identical output.
If there is a specific reason for these additional quotes please let me know!

Reply:
Perhaps @RubenVerborgh knows?

Reply:
@Dexagod But careful, the input you should give the SPARQL validator is `console.log(sourceString)`, not the source code itself, as JavaScript interprets the escapes.
The resulting "`bELECT`" query will fail.